x-pack/plugins/enterprise_search/public/applications/shared/get_cloud_enterprise_search_host/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
```

Comment:
To improve typescript performance, we don't do barrel files anymore. this should be deleted and imports pointing to here should be adjusted.